lib/new-flatfile.ts

```diff
@@ -0,0 +1,92 @@
+import { FlatfileClient } from "@flatfile/api";
```

Comment:
I renamed the old flatfile file and give the new one the name `flatfile.ts`, you can use that one

Reply:
Edited to cutover to exiting flatfile file